<?xml version="1.0" standalone="yes"?>
<NewDataSet>
  <xs:schema id="NewDataSet" xmlns="" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
    <xs:element name="NewDataSet" msdata:IsDataSet="true" msdata:MainDataTable="Table1" msdata:UseCurrentLocale="true">
      <xs:complexType>
        <xs:choice minOccurs="0" maxOccurs="unbounded">
          <xs:element name="Table1">
            <xs:complexType>
              <xs:sequence>
                <xs:element name="Error_Type" type="xs:string" minOccurs="0" />
                <xs:element name="Error_DateTime" type="xs:dateTime" minOccurs="0" />
                <xs:element name="Error_Description" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Source" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Stack_Trace" type="xs:string" minOccurs="0" />
                <xs:element name="Error_Caused_By_Method" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_1" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_2" type="xs:string" minOccurs="0" />
                <xs:element name="Optional_Text_3" type="xs:string" minOccurs="0" />
                <xs:element name="NombreUsuario" type="xs:string" minOccurs="0" />
                <xs:element name="HostName" type="xs:string" minOccurs="0" />
                <xs:element name="IPAddress" type="xs:string" minOccurs="0" />
              </xs:sequence>
            </xs:complexType>
          </xs:element>
        </xs:choice>
      </xs:complexType>
    </xs:element>
  </xs:schema>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-21T15:46:00.6553799-05:00</Error_DateTime>
    <Error_Description>La columna 'Proyecto' no pertenece a la tabla .</Error_Description>
    <Error_Source>System.Data</Error_Source>
    <Error_Stack_Trace>   en System.Data.DataRow.GetDataColumn(String columnName)
   en System.Data.DataRow.get_Item(String columnName)
   en AplicacionSistemaVentura.PAQ01_Definicion.GestProyectos.LlenarDatos() en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\PAQ01-Definicion\GestProyectos.xaml.cs:línea 261</Error_Stack_Trace>
    <Error_Caused_By_Method>System.Data.DataColumn GetDataColumn(System.String)</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-21T16:05:31.3052715-05:00</Error_DateTime>
    <Error_Description>Procedure or function 'Actividad_Update' expects parameter '@FlagActivo', which was not supplied.</Error_Description>
    <Error_Source>.Net SqlClient Data Provider</Error_Source>
    <Error_Stack_Trace>   en System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   en System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean&amp; dataReady)
   en System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   en System.Data.SqlClient.SqlDataReader.get_MetaData()
   en System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task&amp; task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task&amp; task, Boolean&amp; usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteScalar()
   en Data.D_Proyectos.Proyectos_Update(E_Proyectos obje) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\Data\D_Proyectos.cs:línea 27
   en Business.B_Proyectos.Proyectos_Update(E_Proyectos E_Proyectos) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\Business\B_Proyectos.cs:línea 25
   en AplicacionSistemaVentura.PAQ01_Definicion.GestProyectos.BtnGrabar_Click(Object sender, RoutedEventArgs e) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\PAQ01-Definicion\GestProyectos.xaml.cs:línea 86</Error_Stack_Trace>
    <Error_Caused_By_Method>Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-21T16:06:13.9684885-05:00</Error_DateTime>
    <Error_Description>Procedure or function 'Actividad_Update' expects parameter '@FlagActivo', which was not supplied.</Error_Description>
    <Error_Source>.Net SqlClient Data Provider</Error_Source>
    <Error_Stack_Trace>   en System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   en System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean&amp; dataReady)
   en System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   en System.Data.SqlClient.SqlDataReader.get_MetaData()
   en System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task&amp; task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task&amp; task, Boolean&amp; usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteScalar()
   en Data.D_Proyectos.Proyectos_Update(E_Proyectos obje) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\Data\D_Proyectos.cs:línea 27
   en Business.B_Proyectos.Proyectos_Update(E_Proyectos E_Proyectos) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\Business\B_Proyectos.cs:línea 25
   en AplicacionSistemaVentura.PAQ01_Definicion.GestProyectos.BtnGrabar_Click(Object sender, RoutedEventArgs e) en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\PAQ01-Definicion\GestProyectos.xaml.cs:línea 86</Error_Stack_Trace>
    <Error_Caused_By_Method>Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-21T21:01:02.0669923-05:00</Error_DateTime>
    <Error_Description>No hay ninguna fila en la posición 0.</Error_Description>
    <Error_Source>System.Data</Error_Source>
    <Error_Stack_Trace>   en System.Data.RBTree`1.GetNodeByIndex(Int32 userIndex)
   en System.Data.DataRowCollection.get_Item(Int32 index)
   en AplicacionSistemaVentura.PAQ01_Definicion.GestTipoLLantas.LlenarDatos() en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\PAQ01-Definicion\GestTipoLLantas.xaml.cs:línea 275</Error_Stack_Trace>
    <Error_Caused_By_Method>NodePath GetNodeByIndex(Int32)</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-21T21:06:54.6967283-05:00</Error_DateTime>
    <Error_Description>La columna 'CodPro' no pertenece a la tabla .</Error_Description>
    <Error_Source>System.Data</Error_Source>
    <Error_Stack_Trace>   en System.Data.DataRow.GetDataColumn(String columnName)
   en System.Data.DataRow.get_Item(String columnName)
   en AplicacionSistemaVentura.PAQ01_Definicion.GestTipoLLantas.LlenarDatos() en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\PAQ01-Definicion\GestTipoLLantas.xaml.cs:línea 276</Error_Stack_Trace>
    <Error_Caused_By_Method>System.Data.DataColumn GetDataColumn(System.String)</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
  <Table1>
    <Error_Type>System.Collections.ListDictionaryInternal</Error_Type>
    <Error_DateTime>2021-02-21T21:20:50.6064242-05:00</Error_DateTime>
    <Error_Description>Índice fuera de los límites de la matriz.</Error_Description>
    <Error_Source>AplicacionSistemaVentura</Error_Source>
    <Error_Stack_Trace>   en AplicacionSistemaVentura.InterfazPrincipal.VerificarAlertas() en C:\Users\gparedes\Desktop\VSMONE\VSMONE_FUENTE\HANA\AplicacionSistemaVentura\InterfazPrincipal.xaml.cs:línea 257</Error_Stack_Trace>
    <Error_Caused_By_Method>Void VerificarAlertas()</Error_Caused_By_Method>
    <Optional_Text_1 />
    <Optional_Text_2 />
    <Optional_Text_3 />
    <NombreUsuario>MANAGER</NombreUsuario>
    <HostName>WIN-Q3O0BSPVASQ</HostName>
    <IPAddress>192.168.1.40</IPAddress>
  </Table1>
</NewDataSet>